src/components/SatelliteManifestPanel/satelliteManifestPanelUtils.tsx

```diff
@@ -113,14 +112,34 @@ export const sortFilteredRows = (
 ): ManifestRow[] => {
   const directionFactor = sortDirection === SortByDirection.desc ? -1 : 1;
   const sortedRows = filteredRows.sort((a: ManifestRow, b: ManifestRow) => {
-    const term1 = a[sortKey].toLowerCase();
-    const term2 = b[sortKey].toLowerCase();
-    if (term1 < term2) {
-      return -1 * directionFactor;
-    } else if (term1 > term2) {
-      return 1 * directionFactor;
+    if (sortKey == 'version') {
+      let aVersion = a.version;
+      let bVersion = b.version;
+      aVersion = aVersion
+        .split('.')
+        .map((n) => +n + 100000)
+        .join('.');
+      bVersion = bVersion
+        .split('.')
+        .map((n) => +n + 100000)
+        .join('.');
+      if (aVersion < bVersion) {
```

Comment:
This logic is the same as the below logic, and could probably be combined. Psuedocode below for reference

```
let aVal;
let bVal;
if (sortKey is version) {
  aVal = fixedVersionForComparison
  bVal = fixedVersionForComparison
} else {
  aVal = a[sortKey].toLowerCase();
  bVal = b[sortKey].toLowerCase();
}

// compare aVal and bVal
```

Reply:
The only logic that changes based on whether its version or not is what the values are set to